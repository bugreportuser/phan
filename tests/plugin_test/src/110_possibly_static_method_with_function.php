<?php
class PSMClazz110 {
    public function main() {
        function inner_global_function() {
            var_export($this);
        }
        inner_global_function();
    }
}
(new PSMClazz110())->main();
