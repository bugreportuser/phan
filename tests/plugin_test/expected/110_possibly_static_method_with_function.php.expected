src/110_possibly_static_method_with_function.php:3 PhanPluginPossiblyStaticPublicMethod Public method \PSMClazz110::main can be static
src/110_possibly_static_method_with_function.php:5 PhanUndeclaredVariable Variable $this is undeclared
